targets/TARGET_STM/TARGET_STM32F0/serial_device.c

```diff
@@ -87,47 +93,46 @@ static void uart2_irq(void)
 }
 #endif
 
-#if defined USART3_BASE
-static void uart3_irq(void)
+// Used for both USART3_4_IRQn and USART3_8_IRQn
+static void uart3_8_irq(void)
 {
-    uart_irq(2);
-}
+#if defined(TARGET_STM32F091RC)
```

Comment:
Does this problem only affect the STM32F091RC, or are more targets affected?

Reply:
All STM32F09x devices support this but there is only the STM32F091RC device on mbed actually.